spec/apruve/resources/order_spec.rb

```diff
@@ -302,10 +302,10 @@
     end
 
     describe '#payment_term' do
-      let (:order) { Apruve::Order.new id: id, merchant_id: 9999, payment_terms: payment_term }
+      let (:order) { Apruve::Order.new id: id, merchant_id: 9999, payment_term: payment_term }
 
       it 'returns payment_terms' do
-        expect(order.payment_term).to be order.payment_terms
+        expect(order.payment_term).to be order.payment_term
```

Comment:
Isn't this doing nothing?
Don't you want `expect(order.payment_term).to be payment_term`

Reply:
fixed